src/client/src/orangehrmAuthenticationPlugin/components/SocialMediaAuth.vue

```diff
@@ -30,6 +30,13 @@
       :url="authenticator.url"
       :color="authenticator.color"
       :label="authenticator.label"
+      :style="{
+        background:
```

Comment:
can you attach a screenshot of the login screen with this images

Reply:
![Uploading Screenshot from 2023-12-15 09-21-23.png…]()

Reply:
Is the screenshot visible? showing as broken image for me 
![image](https://github.com/orangehrm/orangehrm/assets/97493184/8d2ff349-9195-4985-812b-0d4705b294a1)